test/dlc_tests/eks/mxnet/training/test_eks_mxnet_training.py

```diff
@@ -99,6 +96,7 @@ def test_eks_mxnet_dgl_single_node_training(mxnet_training, py3_only):
     cpu_limit = str(int(cpu_limit) / 2)
 
     if "gpu" in mxnet_training:
+        pytest.skip("Skipping DGL tests for GPU until dgl-cu110 is available.")
```

Comment:
Please check for `cu110` in the image tag, rather than just skipping for all GPU images, so that older framework versions can still be tested.